VideoLocker/src/main/java/org/edx/mobile/util/Config.java

```diff
@@ -228,6 +233,18 @@ public String getNewRelicKey() {
     }
 
     /**
+     * End to end test configurations
+     */
+    public class EndToEndConfig {
+        private String DEFAULT_EMAIL_TEMPLATE = "test-{unique_id}@example.com";
+        private @SerializedName("EMAIL_TEMPLATE") String mEmailTemplate;
```

Comment:
Since you're adding the end-to-end testing config, I guess the `TEST_COURSE_ID` key should also be defined here, even though we haven't added the test for it yet.